index.html

```diff
@@ -219,6 +219,91 @@ <h3>Skills</h3>
 				</div>
 			</div>
 		</section>
+		<section id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
+					<label class="firstN"
+						><input
+							type="text"
+							name="firstN"
+							id="firstN"
+							placeholder="First name"
+							required
+						/>
+					</label>
+					<label class="lastN">
+						<input
+							type="text"
+							name="lastN"
+							id="lastN"
+							placeholder="Last Name"
+							required
+						/>
+					</label>
+					<label class="fullN"
+						><input
+							type="text"
+							name="fullname"
+							id="fullname"
+							placeholder="Full name"
+							required
+						/>
```

Comment:
- As suggested by the previous reviewer it would be  great if we stick to the project requirement as much as possible. In the project requirement it was stated that we give the name input form a property of `maxlength=30`. But currently its missing in your name input form, kindly add it